app.json

```diff
@@ -1,4 +1,10 @@
 {
   "name": "ReactNativeBase",
-  "displayName": "ReactNativeBase"
-}
\ No newline at end of file
+  "displayName": "ReactNativeBase",
+  "expo": {
+    "extra": {
+      "eas": {
+      }
```

Comment:
Should we add the `projectId` placeholder here?

Reply:
It is added automatically when you execute the EAS build command and add the project to your Expo account.